src/config/index.ts

```diff
@@ -36,15 +38,18 @@ const rawConfig: RawConfig = {
 }
 
 if (!rawConfig.auth.driverName) {
-	console.warn(`No auth driver selected.
-		A default BearerToken driver is selected and a random bearer token will be generated.`)
+	logger.warn(`No auth driver selected.
+	    A default BearerToken driver is selected and a random bearer token will be generated.`)
 
 	rawConfig.auth.driverName = AuthDriverName.BearerToken
 	rawConfig.auth.bearerTokenDriverConfig.token = cryptoRandomString({ length: 18, type: 'alphanumeric' })
 	console.info(`Generated bearer token is '${rawConfig.auth.bearerTokenDriverConfig.token}'`)
 }
 
-if (!validateConfig(rawConfig)) throw new InvalidConfigError()
+if (!validateConfig(rawConfig)) {
+	logger.error('rawConfig is not validated')
```

Comment:
```suggestion
	logger.error('Recieved config is invalid')
```

These logs will be seen by users. They don't know what RawConfig means.